ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -163,6 +163,16 @@ objects:
         value: ${IMAGE_SYNC}
       - name: TEST_IN_PRODUCTION
         value: ${TEST_IN_PRODUCTION}
+      - name: CODECOV_UPLOAD_TOKEN
```

Comment:
Why mount the upload token in the e2e template when it's not going to be used here?